assets/mod_info.json

```diff
@@ -3,7 +3,7 @@
     "name": "Stellar Networks",
     "author": "Jaghaimo",
     "utility": true,
-    "version": "3.2.0",
+    "version": "3.2.1",
     "description": "A collection of intel boards for Starsector",
     "gameVersion": "0.98a-RC5",
```

Comment:
There was a new version released, we could bump the version to RC7 if we update dependency in the api repo, but not crucial (change tested with both RC5 and RC7).

Reply:
API has not changed, only implementation. Safe to build using RC5. I'll update API repo to RC7 today.

Reply:
If you want to build as-is, then resolve all and merge. Otherwise wait till I update to RC7.

Reply:
Updated to RC7

Reply:
cheers, updated